lib/matplotlib/tri/_trirefine.py

```diff
@@ -204,8 +204,8 @@ def _refine_triangulation_once(triangulation, ancestors=None):
             if np.shape(ancestors) != (ntri,):
                 raise ValueError(
                     "Incompatible shapes provide for triangulation"
-                    ".masked_triangles and ancestors: {0} and {1}".format(
-                        np.shape(triangles), np.shape(ancestors)))
+                    ".masked_triangles and ancestors: "
+                    f"{np.shape(triangles)} and {np.shape(ancestors)}")
```

Comment:
We can re-wrap a little now to avoid the break in the middle of an expression:
```suggestion
                    "Incompatible shapes provide for "
                    "triangulation.masked_triangles and ancestors: "
                    f"{np.shape(triangles)} and {np.shape(ancestors)}")
```